lib/src/builder_implementation.dart

```diff
@@ -4320,6 +4327,7 @@ String _extractConstantCode(
         Element staticElement = expression.staticElement;
         if (staticElement is PropertyAccessorElement) {
           VariableElement variable = staticElement.variable;
+          // ignore:deprecated_member_use
```

Comment:
Perhaps add a TODO together with these to indicate that there is more work.

Reply:
Done.